README.md

```diff
@@ -87,3 +88,7 @@ Examples
 | .SomeComponent-isActive | X | | X |
 | .SomeComponent_Navigation-next | X | X | X |
 | .SomeComponent_NavigationControls-invalidError | X | X | X |
+
+#### ModalStack
```

Comment:
I think this should probably go in a read me in `packages/react-ui-core` instead.  We should probably move the react-themed stuff there as well.

Reply:
react-ui-core doesn't have it's own top-level README yet. Adding one just for this one link didn't makes sense to me, but if you prefer I can add one.